UserStoryDemo/src/components/atoms/Button/index.tsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react'
+import Button, { ButtonProps as MuiButtonProps } from '@mui/material/Button'
+
+interface ExtendedButtonProps extends MuiButtonProps {
+  startIcon?: React.ReactNode

```

Comment:
no need to again override the startIcon and endIcon , as you are extending the base MuiButtonProps those all will be present , and can be passed from where the Button compo is called